Book/php7/internal_types/functions.rst

```diff
@@ -1,4 +1,15 @@
-Functions: zend_function
+Functions:
 ========================
 
-Here we detail how functions work in PHP.
+The body of PHP functions are represented with the ``zend_function`` structure.
+However, handling them is rarely done as they are solely needed for the PHP VM.
+In general PHP ``callable``s are what will need to be dealt with, which are represented by the pair of
```

Comment:
Apparently, Sphinx doesn't understand the closing double-backticks when directly followed by a letter. Maybe insert a space (there might be better solution, though):
```suggestion
In general PHP ``callable`` s are what will need to be dealt with, which are represented by the pair of
```